packages/components/src/tooltip/src/Tooltip.tsx

```diff
@@ -41,20 +43,27 @@ export function InnerTooltip({
     });
 
     return (
-        <Text
+        <Div
             {...mergeProps(
                 rest,
                 {
                     as,
-                    className: "o-ui-tooltip",
```

Comment:
Changing the structure of the tooltip break 6 chromatic test. Before we were able to modify the border color of the tooltip for instance. It was broken because the style was not propagated to the arrow. 

Now the style is forwarded to the wrapper, so the border doesn't target the tooltip's border. 

I think that it's ok and that i don't need to change anything

Reply:
should forward rest to the tooltip not the wrapper